docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+services:
+  autorag:
+    build:
+      context: .
+      dockerfile: Dockerfile
+      target: production
+    volumes:
+      - ~/.cache/huggingface:/cache/huggingface
+      - ./sample_config:/usr/src/app/sample_config
+      - ./projects:/usr/src/app/projects
+    environment:
+      HF_HOME: /cache/huggingface
+      OPENAI_API_KEY: ${OPENAI_API_KEY}  # Include the OpenAI API key as an environment variable
+    command: >
+      validate --config /usr/src/app/sample_config/rag/simple/simple_openai.yaml
+      --qa_data_path /usr/src/app/projects/test01/qa_validation.parquet
+      --corpus_data_path /usr/src/app/projects/test01/corpus.parquet
```

Comment:
Is this run the specific config file, qa dataset, corpus dataset as command?
I think it is okay not to give it at the `docker-compose.yml` isn't it?
Is there a reason to set the exact command at docker compose?

Reply:
For beginners, we should have a clear guide on how to use this project. I propose defining a folder structure that includes a dedicated dataset download folder. This way, after downloading the dataset, we can provide a starter configuration that works without requiring an OpenAI Key. I want to ensure they have a smooth experience right after cloning the repository.